coco/master.py

```diff
@@ -165,12 +223,14 @@ def _load_config(self, config_path):
 
         self.log_level = config.get("log_level", "INFO")
         self.endpoint_dir = config["endpoint_dir"]
+        self.prom_dir = config["prometheus_multiproc_dir"]
         try:
             self.slack_url = config["slack_webhook"]
         except KeyError:
             self.slack_url = None
             logger.warning("Config variable 'slack_webhook' not found. Slack messaging DISABLED.")
         self.port = config["port"]
+        self.worker_port = config["worker_port"]
```

Comment:
Let's change this name to something more obvious! How about `"worker_metrics_port"`?